source/feature/feature-template.html.md

```diff
@@ -55,7 +61,7 @@ Describe user experience related issues. For example: We need a wizard for ....,
 
 #### Installation/Upgrade
 
-Describe how the feature will effect new installation or existing one.
+Describe how this feature will affect new installations or existing ones.
```

Comment:
Suggest to change to:
"Describe how this feature will affect new or existing installations."